eslint/react/.eslintrc

```diff
@@ -18,13 +18,64 @@
 	},
 	"rules": {
 		"class-methods-use-this": "off",
+		"jsx-quotes": "error",
+		"react/boolean-prop-naming": "error",
+		"react/button-has-type": "error",
+		"react/default-props-match-prop-types": "error",
+		"react/jsx-boolean-value": ["error", "never"],
+		"react/jsx-child-element-spacing": "error",
```

Comment:
this one is supper weird and it never happened to me

Reply:
I agree it's a super edgy case, but there's no harm in having it covered, no?

Reply:
i find it super ugly if the linter forces you to add spacing in this way, i would add it when necessary

Reply:
the way to fix it is super ugly, i don't know